lib/GADS/Filter.pm

```diff
@@ -120,24 +124,25 @@ sub _clear_lazy
 sub base64
 {   my $self = shift;
     # First make sure we have the hash version
+    return encode_base64($self->as_json, '') if $self->people_filter; #If we do the below on a people filter it dies
     $self->as_hash;
     # Then clear the JSON version so that we can rebuild it
     $self->clear_as_json;
     # Next update the filters
-    foreach my $filter (@{$self->filters})
-    {
-        $self->layout or panic "layout has not been set in filter";
-        my $col = $self->layout->column($filter->{column_id})
-            or next; # Ignore invalid - possibly since deleted
-        if ($col->has_filter_typeahead)
-        {
-            $filter->{data} = {
-                text => $col->filter_value_to_text($filter->{value}),
-            };
-        }
-        if ($col->type eq 'filval')
-        {
-            $filter->{filtered} = $col->related_field_id,
+    if(@{$self->filters}) {
```

Comment:
Is this condition necessary? The loop before would not be run anyway in such circumstances

Reply:
I believe this was for a slightly different implementation I previously tried that didn't work - now removed. :smile: